From 3eb43e2e9df80baad20722b3698e6995a546853a Mon Sep 17 00:00:00 2001
From: Carlo Landmeter <clandmeter@alpinelinux.org>
Date: Tue, 29 Jun 2021 11:56:32 +0000
Subject: [PATCH] ffbuild/libversion.sh: add shebang

when using something like qemu-user where argv[0] is qemu the script
will not be exexuted by the shell.
---
 ffbuild/libversion.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/ffbuild/libversion.sh b/ffbuild/libversion.sh
index 990ce9f640..30046b1d25 100755
--- a/ffbuild/libversion.sh
+++ b/ffbuild/libversion.sh
@@ -1,3 +1,5 @@
+#!/bin/sh
+
 toupper(){
     echo "$@" | tr abcdefghijklmnopqrstuvwxyz ABCDEFGHIJKLMNOPQRSTUVWXYZ
 }
-- 
2.32.0

